### PR TITLE
Added installation method for npm

### PR DESCRIPTION
**Pull request by Prashoon123**
## Summary
In this PR I have added the installation method for npm.

## Test Plan
image

### CODE REVIEW

File: website/docs/Installation.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ sidebar_label: Installation
 ### Get library
 
 ```bash
+// with npm
```
Comment (Krizzu) [inline website/docs/Installation.md:12]: comments in bash are started by `#`

**Comment by Prashoon123** [comment]
@Krizzu I did that change!👍🏻

**Review by tido64 — APPROVED** [review]

**Comment by tido64** [comment]
@Prashoon123 Thanks ❤️

**Comment by Prashoon123** [comment]
You're welcome @tido64!!

**Pull request merged**